c/ldv-linux-3.14-races/README.txt

```diff
@@ -0,0 +1,20 @@
+Verification tasks from Linux kernel 3.14.
+Tasks were taken for concurrency check.
+Contributed by the LDV project.
+
+Scenario for obtaining the tasks.
+1. Download and install Klever tool (http://forge.ispras.ru/projects/klever)
+2. Download and unpack the Linux kernel v3.14  (https://www.kernel.org/pub/linux/kernel/v3.x/linux-3.14.tar.gz)
+3. Prepare job description:
+  3.1 Set the Linux kernel version to 'linux-3.14'
+  3.2 Choose a required module: drivers/media/platform/marvell-ccic/cafe_ccic.ko, drivers/net/irda/nsc-ircc.ko, drivers/net/irda/w83977af_ir.ko, drivers/spi/spi-tegra20-slink.ko, drivers/usb/misc/adutux.ko, drivers/usb/misc/iowarrior.ko.
+  3.3 Set rule specification to 'linux:kernel:locking:svcomp'
+  3.4 The verifier is not neccessary to change, because we do not interested in verification
```

Comment:
Nit picking: "The verifier need not be changed, because we do not perform verification at this point."